README.md

```diff
@@ -130,39 +112,44 @@ function update(state, action) {
 }
 ```
 
-For more context, check out this [full example](https://github.com/freeqaz/redux-simple-router-example).
+### <a id="examples" href="#examples">Examples</a>
+
+* [examples/basic](https://github.com/jlongster/redux-simple-router/blob/master/examples/basic) - basic reference implementation
+
+Examples from the community:
+
+* [davezuko/react-redux-starter-kit](https://github.com/davezuko/react-redux-starter-kit) - popular redux starter kit
+  * **tip**: migrating from redux-router? use [this commit](https://github.com/davezuko/react-redux-starter-kit/commit/db66626ca8a02ecf030a3f7f5a669ac338fd5897) as a reference
+* [freeqaz/redux-simple-router-example](https://github.com/freeqaz/redux-simple-router-example) - example implementation
+
+_Have an example to add? Send us a PR!_
+
+### <a id="api" href="#api">API</a>
+
+#### <a id="syncreduxandrouter" href="#syncreduxandrouter">`syncReduxAndRouter(history, store, selectRouterState?)`</a>
+
+Call this with a react-router and a redux store instance to install hooks that always keep both of them in sync. When one changes, so will the other.
+
+Supply an optional function `selectRouterState` to customize where to find the router state on your app state. It defaults to `state => state.routing`, so you would install the reducer under the name "routing". Feel free to change this to whatever you like.
 
-## API
+#### <a id="routereducer" href="#routereducer">`routeReducer`</a>
 
-### `syncReduxAndRouter(history, store, selectRouterState?)`
+A reducer function that keeps track of the router state. You need to add this reducer to your app reducers when creating the store. **The piece of state must be named `routing`** (unless you provide a custom `selectRouterState` function).
```

Comment:
Maybe change the last part here to this?

> If you don't provide a custom `selectRouterState` function, the piece of state must be named `routing`.

Not sure, just feels strange to highlight this right after "Feel free to change this to whatever you like" in the docs ;)